service/src/test/java/bio/terra/workspace/app/controller/FolderApiControllerTest.java

```diff
@@ -280,8 +283,20 @@ public void deleteFolders_deleteTopLevelFolder_folderAndSubFoldersAllDeleted() t
     ApiFolder thirdFolder =
         createFolder(workspaceId, /*displayName=*/ "foo", /*parentFolderId=*/ secondFolder.getId());
 
-    deleteFolderExpectCode(workspaceId, firstFolder.getId(), HttpStatus.SC_NO_CONTENT);
-
+    var serializedResponse =
+        deleteFolderExpectCode(workspaceId, firstFolder.getId(), HttpStatus.SC_ACCEPTED)
```

Comment:
nit put 287-297 into helper method like deleteFolderAndWaitForJob. It makes test easier to scan.